packages/fxa-settings/src/pages/Signin/SigninBounced/index.tsx

```diff
@@ -2,118 +2,149 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import React from 'react';
-import { RouteComponentProps, useNavigate } from '@reach/router';
-import { usePageViewEvent } from '../../../lib/metrics';
+import React, { useEffect } from 'react';
+import { RouteComponentProps /*useNavigate*/ } from '@reach/router';
+import { usePageViewEvent, logViewEvent } from '../../../lib/metrics';
 import { ReactComponent as EmailBounced } from './graphic_email_bounced.svg';
 import { FtlMsg } from 'fxa-react/lib/utils';
-import { useFtlMsgResolver } from '../../../models/hooks';
+import { useFtlMsgResolver, useWindowWrapper } from '../../../models/hooks';
+
 import LinkExternal from 'fxa-react/components/LinkExternal';
+import AppLayout from '../../../components/AppLayout';
 import { REACT_ENTRYPOINT } from '../../../constants';
 import CardHeader from '../../../components/CardHeader';
+import LoadingSpinner from 'fxa-react/components/LoadingSpinner';
 
 export type SigninBouncedProps = {
+  email?: string;
+  emailLookupComplete: boolean;
   onBackButtonClick?: (
     event: React.MouseEvent<HTMLButtonElement, MouseEvent>
   ) => void;
   canGoBack?: boolean;
-  email: string;
 };
 
 export const viewName = 'signin-bounced';
 
 const SigninBounced = ({
-  canGoBack,
   email,
-  onBackButtonClick = () => window.history.back(),
+  emailLookupComplete,
+  canGoBack,
+  onBackButtonClick,
 }: SigninBouncedProps & RouteComponentProps) => {
   usePageViewEvent(viewName, REACT_ENTRYPOINT);
   const ftlMessageResolver = useFtlMsgResolver();
   const backText = ftlMessageResolver.getMsg('back', 'Back');
-  const navigate = useNavigate();
+  // const navigate = useNavigate();
+
+  const handleNavigationBack = (event: any) => {
+    logViewEvent(viewName, 'link.back', REACT_ENTRYPOINT);
+    if (onBackButtonClick) {
+      onBackButtonClick(event);
+    } else {
+      window.history.back();
+    }
+  };
+
+  useEffect(() => {
+    if (emailLookupComplete && !email) {
+      // TODO: Going from react page to non-react page will require a hard
+      // navigate. When signin flow has been converted we should be able
+      // to use `navigate`
+      // navigate('/signin?showReactApp=true', { replace: true });
+      window.location.replace('/signin');
+    }
+  }, [email, emailLookupComplete /*, navigate*/]);
 
   const createAccountHandler = () => {
-    // TO-DO (per content-server functionality):
-    // clear all existing accounts
-    // clear the session
-    // clear the form prefill
-    navigate('/signup');
+    logViewEvent(viewName, 'link.create-account', REACT_ENTRYPOINT);
```

Comment:
I think this is a separate issue (because I've spotted this in other pages while working on FXA-7100), but the emitted metrics event doesn't have a payload.